gwdetchar/tests/test_plot.py

```diff
@@ -38,6 +38,23 @@
                        name='X1:TEST-FLAG:1')
 
 
+# -- test utilities -----------------------------------------------------------
+
+def test_texify():
+    name = 'X1:TEST-CHANNEL_NAME'
+
+    # test with LaTeX
+    rcParams['text.usetex'] = True
```

Comment:
I think you should use [`mock.patch.dict`](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch.dict) here so that you aren't manipulating the global `rcParams` dict, which could lead to race-conditions.

Reply:
Done in the latest commit, this is a neat trick, thanks for teaching me something!